windows-installer/construct.py

```diff
@@ -98,32 +99,6 @@ def _copy_assets(build_path, repo_path):
 def _generate_pynsist_config(repo_path, build_path):
     print('Generate pynsist configuration')
 
-    pywin32_paths_file = os.path.join(build_path, 'pywin32_paths.py')
```

Comment:
Can you say any more about this change here? How did you discover that this code either no longer works or is no longer needed? What changed? 

I'm glad to see we're simplifying code and the tests pass, but I don't actually immediately understand what's going on here and looking back at [the PR that added this code in the first place](https://github.com/certbot/certbot/pull/7377) didn't really help me.

Reply:
Ah yes. Some history about this. It is related to the capacity to import `pywintypes` and `pythoncom`, which are precompiled DLLs that contains symbol and mapping for the data exchanged through the Windows kernel API.

The way the modules are loaded in an application built with pynsist is quite specific because the site-package used to install the application is not very standard. You can see the loader of `sys.path` here: https://github.com/takluyver/pynsist/blob/89431e7c382da103f08c2fe006e388db4d2cf09b/nsist/commands.py#L8-L26

Basically it searches every module in the `pkgs` folder in `INSTALLDIR`, which acts as a "site-package" folder.

Initially with this approach the Python interpreter was not capable to find the DLLs for `pywintypes` and `pythoncom`. The initial script to solve that was taken from the `pynsist` project itself (https://github.com/takluyver/pynsist/blob/master/examples/pywebview/pywin32_paths.py). It was added as a preample script added by pynsist just after the standard `sys.path` loader.

It seems that now this resolution has been integrated in the `pywin32` project. I did not dig enough to fully understand the modifications, but I find a similar machinery for these DLL load under various files installed in the `pkg` folder by `pywin32` (in particular `win32\lib\pywintypes.py` and `pythoncom.py`). Now the old preamble is breaking: it seems that the "let pywin32 does its magic" is now the best approach, and makes a fully functional Certbot as the integration tests show that.